src/collector.ts

```diff
@@ -100,6 +100,7 @@ if (config.env == envEnum.DEV) {
 let rmqCyclesConsumer: RMQCyclesConsumer
 let rmqTransactionsConsumer: RMQOriginalTxsConsumer
 let rmqReceiptsConsumer: RMQReceiptsConsumer
+let stopCheckpointVerificationProcess = false
```

Comment:
Introduced this variable to help get out of the forever running verification loop when exit signals are received.